matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/RoundedRectangleImageView.java

```diff
@@ -16,42 +16,98 @@
 package com.zhihu.matisse.internal.ui.widget;
 
 import android.content.Context;
+import android.content.res.Configuration;
+import android.graphics.Bitmap;
 import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
 import android.graphics.Path;
+import android.graphics.PorterDuff;
+import android.graphics.PorterDuffXfermode;
 import android.graphics.RectF;
+import android.graphics.drawable.Drawable;
 import android.support.v7.widget.AppCompatImageView;
 import android.util.AttributeSet;
+import android.util.TypedValue;
 
 public class RoundedRectangleImageView extends AppCompatImageView {
```

Comment:
@unixzii Actually I think the original implementation is right, and I have not found the bug···Maybe you can see this [RoundedImageView](https://github.com/vinc3m1/RoundedImageView).